### DISCUSSION TITLE
Change authority account from admin

### DISCUSSION THREAD

**Discussion by rimzzlabs** [umami-software/umami#1616] [Q&A] [answered]
Hi, I have a question, currently I have umami but I made a mistake because I've linked my website to my umami **admin** account.

Im a frontend dev so I don't know if I'm able to move my current all datas from admin account to a new account on umami?

**Comment by rimzzlabs** [comment, marked as answer]
turns out I can change the relational of my website to another account on the database😄

**Reply by MagicLike** [reply]
You can just rename the admin account and use that as your account
